# gets the current temperature in degrees Celsius from the humidity sensor
---
float64 temperature